manifests/charts/istiod-remote/files/gen-istiod-remote.yaml

```diff
@@ -676,7 +675,10 @@ metadata:
 webhooks:
   - name: sidecar-injector.istio.io
     clientConfig:
-      url: https://istiod.istio-system.svc:443/inject
+      service:
```

Comment:
this looks wrong, do you need to update istiod-remote/values.yaml to use the `injectionURL`?

Reply:
Hmm.. looks like you did but its not actually applying from what I can tell? I would expect this to be `url:`

Reply:
yes you do as default value is empty.  This generated file is based on the default value which is when injection url isn't specified and it would be using in cluster service.